bridgebot/bidding.py

```diff
@@ -75,16 +71,16 @@ def __increment_player(self):
 
     def pass_bid(self):
         if self.bid_index == -1:
-            if self.consecutive_passes == 3: # Passing out
-                self.record[self.player].append(PASS)
+            if self.consecutive_passes == 3:  # Passing out
+                self.record[self.player].append(Strains.PASSOUT)
                 self.contract.strain = Strains.NT
                 return AuctionStatus.DONE
             else:  # Passing before an opening bid
-                self.record[self.player].append(PASS)
+                self.record[self.player].append(Strains.PASSOUT)
```

Comment:
Not sure if this is correct. The point is to keep track of what the players bid, not the final strain.